OracleHTTPServer/dockerfiles/12.2.1.2.0/Dockerfile

```diff
@@ -67,21 +70,39 @@ RUN yum install -y \
         gcc gcc-c++ glibc glibc-devel libaio libaio-devel libgcc \
 	libstdc++ libstdc++-devel ksh make ocfs2-tools sysstat && \
```

Comment:
Do you really need `ocfs2-tools`? There is no way to mount an OCFS2 filesystem from inside a container.